### PR TITLE
converted to dotnet standard 2.0.

### PR DESCRIPTION
**Pull request by MingLu8**
Hi Luiz

I converted this package to use dotnet standard 2.0 using System.Drawing.Common, it seems like all the tests passed.  And I incremented the nuget package version to 2.0.0 using the csproj file assembly version and remove .nuspec file.

Could you please check it out and merge it?

Thanks

Ming

### CODE REVIEW

**Comment by UlyssesWu** [comment]
Sorry but I'm voting down for this because you didn't handle for platforms other than Windows. Therefore there is almost meaningless for netstandard. 

I've made a maybe better port before you did:
https://github.com/UlyssesWu/FastBitmap

with a nuget package on myget:
https://www.myget.org/feed/monarchsolutions/package/nuget/FastBitmapLib

I didn't make a PR because I haven't got time to test it on other platforms.

File: FastBitmap/FastBitmapLib.csproj [hunk 1]
```diff
@@ -1,57 +1,25 @@
...
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <TargetFramework>netstandard2.0</TargetFramework>
     <RootNamespace>FastBitmapLib</RootNamespace>
     <AssemblyName>FastBitmapLib</AssemblyName>
-    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Version>2.0.0</Version>
+    <Authors>ASI Architecture Team</Authors>
```
Comment (UlyssesWu) [inline FastBitmap/FastBitmapLib.csproj:9]: I assume the author should be Luiz Fernando .

File: FastBitmap/FastBitmap.cs [hunk 2]
```diff
@@ -760,19 +760,19 @@ public static Bitmap SliceBitmap(Bitmap source, Rectangle region)
 
             return slicedBitmap;
         }
-        
+
         /// <summary>
         /// .NET wrapper to native call of 'memcpy'. Requires Microsoft Visual C++ Runtime installed
         /// </summary>
         [DllImport("msvcrt.dll", EntryPoint = "memcpy", CallingConvention = CallingConvention.Cdecl, SetLastError = false)]
```
Comment (UlyssesWu) [inline FastBitmap/FastBitmap.cs:767]: Have you tried this on Linux? 😁

**Review by UlyssesWu — CHANGES_REQUESTED** [review]

**Comment by LuizZak** [comment]
Hello @MingLu8! I'll be merging @UlyssesWu's PR because it's a bit more comprehensive, but thank you for wanting to contribute either way!

**Pull request closed without merging**